src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberMethodSymbol.cs

```diff
@@ -859,6 +873,12 @@ internal override void AfterAddingTypeMembersChecks(ConversionsBase conversions,
             return builder.MostCommonValue;
         }
 
+        internal override bool IsNullableEnabled()
```

Comment:
>IsNullableEnabled [](start = 31, length = 17)

It feels like this name should be more specific.

Reply:
The method name seems ok to me. Feel free to suggest an alternative.

---
In reply to: [546809723](https://github.com/dotnet/roslyn/pull/49953#discussion_r546809723) [](ancestors = 546809723)

Reply:
> The method name seems ok to me, and seems consistent with the use of IsNullableEnabled elsewhere in the PR.

Different aspect can be enabled for "nullable". For example, helpers in compilation are using "Analysis" in the names: "IsNullableAnalysisEnabled", etc. I think this greatly clarifies the purpose. Consider following the same pattern, i.e. naming this  "IsNullableAnalysisEnabled".

---
In reply to: [549578346](https://github.com/dotnet/roslyn/pull/49953#discussion_r549578346) [](ancestors = 549578346,546809723)